tests/nested/manual/core20-set-efi-boot-vars/task.yaml

```diff
@@ -0,0 +1,206 @@
+summary: Check that EFI boot variables are successfully on UC20+
+
+details: >
+    This test checks that EFI boot variables are correctly set during
+    installation.  In particular, there should be a Boot#### variable with
+    data pointing to either \EFI\ubuntu\shimx64.efi or \EFI\boot\bootx64.efi,
+    depending on the version of the pc-gadget snap, and the first number in the
+    BootOrder variable should be the corresponding number.
+
+systems: [ubuntu-20.04-64, ubuntu-22.04-64]
```

Comment:
Since it's running a nested ubuntu core image, what would the behavior be on 23.04 or 23.10, for example? I figured explicitly listing the systems with corresponding core2* versions was safe, happy to adjust of course.

Reply:
`systems:` for google-nested suite do not include 23.0X, just the LTS releases